libs/localization/src/lib/en.json

```diff
@@ -680,7 +687,13 @@
       "cohortName": "Cohort name"
     },
     "TreeView": {
-      "treeDescription": "To find nodes in the tree map with the most errors, look for higher values and fuller circles."
+      "treeDescription": "The tree visualization uses the mutual information between each feature and the error to best separate error instances from success instances hierarchically in the data.",
+      "treeDescriptionExpandedPar1": "This simplifies the process of discovering and highlighting common failure patterns.",
+      "treeDescriptionExpandedPar2": "To ",
```

Comment:
I don't think this will work. We can't expect the sentence structure to align with this in all languages. Is it a huge issue if there no bold text?

Reply:
talked to PMs, will remove it